wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h

```diff
@@ -42,7 +42,7 @@ typedef enum {
     tsip_Key_RSA1024 = 4,
     tsip_Key_RSA2048 = 5,
     tsip_Key_tls_Rsa2048 = 6,
-    UNKNOW = -1,
```

Comment:
Is "UNKNOW" used anywhere by Renesas?

Reply:
I checked and as far as I can tell its not. @miyazakh can you confirm?

Reply:
We should consider removing the enum if it's not used. Very likely to have conflicts with "UNKNOWN" already defined/used.

Reply:
Sorry. I missed the comment. I will take care of "UNKNOWN".